chemprop/cli/train.py

```diff
@@ -113,7 +113,7 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         "--frzn-ffn-layers",
         type=int,
         default=0,
-        help="Overwrites weights for the first n layers of the ffn from checkpoint model (specified ``model-frzn``), where n is specified in the input (also automatically freezes mpnn weights)",
+        help="Freeze the first n layers (default is 0, no layers frozen) of the FFN from the checkpoint model (specified by ``model-frzn``). This option also freezes all MPNN weights.",
```

Comment:
```suggestion
        help="Freeze the first ``n`` layers of the FFN from the checkpoint model (specified by ``model-frzn``). All MPNN weights are automatically frozen when ``model-frzn`` is used.",
```

Reply:
For clarification, the MPNN weights are overwritten and frozen when `--model-frzn` is used, regardless of the setting of `--frzn-ffn-layers`.

Reply:
Thx, I think then it is better to clarify how MPNN weights are frozen in `--model-frzn` instead. I have changed it.